sale_order_import_invoice2data/README.rst

```diff
@@ -0,0 +1,100 @@
+
+French users should also install the module *l10n_fr_invoice_import* available in the `French localization <https://github.com/OCA/l10n-france/>`_, cf `this PR <https://github.com/OCA/l10n-france/pull/55>`_.
+
+Configuration
+=============
+
+Go to the form view of the supplier and configure it with the following parameters:
+
+* *is a Company ?* is True
+* *Supplier* is True
+* the *TIN* (i.e. VAT number) is set (the VAT number is used by default when searching the supplier in the Odoo partner database)
+* in the *Accounting* tab, create an *Invoice Import Configuration*.
+
+For the PDF invoice of your supplier that don't have an embedded XML file, you will have to create a `template file <https://github.com/m3nu/invoice2data/blob/master/invoice2data/templates>`_ in YAML format in the invoice2data Python library. It is quite easy to do ; if you are familiar with `regexp <https://docs.python.org/2/library/re.html>`_, it should not take more than 10 minutes for each supplier.
+
+Here are some hints to help you add a template for your supplier:
+
+* Take `Free SAS template file <https://github.com/m3nu/invoice2data/blob/master/invoice2data/templates/fr/fr.free.adsl-fiber.yml>`_ as an example. You will find a sample PDF invoice for this supplier under invoice2data/test/pdfs/2015-07-02-invoice_free_fiber.pdf
+
+* Try to run the invoice2data library manually on the sample invoice of Free:
+
+.. code::
+
+  % python -m invoice2data.main --debug invoice2data/test/pdfs/2015-07-02-invoice_free_fiber.pdf
+
+On the output, you will get first the text of the PDF, then some debug info on the parsing of the invoice and the regexps, and, on the last line, you will have the dict that contain the result of the parsing.
+
+* if the VAT number of the supplier is present in the text of the PDF Sale Order, I think it's a good idea to use it as the keyword. It is a good practice to add 2 other keywoards: one for the
+language (for example, match on the word *Invoice* in the language of the invoice) and one for the currency, so as to match only the invoices of that supplier in this particular language and currency.
+
+* the list of *fields* should contain the following entries:
+
+  * 'vat' with the VAT number of the supplier (if the VAT number of the supplier is not in the text of PDF file, add a 'partner_name' key, or, if the supplier is French and the module *l10n_fr_invoice_pdf_import* is installed, add a 'siren' key)
+  * 'amount' ('amount' is the total amount with taxes)
+  * 'amount_untaxed' or 'amount_tax' (one or the other, no need for both)
+  * 'date': the date of the invoice
+  * 'invoice_number'
+  * 'date_due', if this information is available in the text of the PDF file.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Tom Blauwendraat <info@sunflowerweb.nl>
+* Terence Nzaywa <terrence@sunflowerweb.nl>
+* Alexis de Lattre <alexis.delattre@akretion.com>
+
+Maintainer
+----------
+
+This module is maintained by Sunflower IT and Therp BV.
```

Comment:
for OCA modules, it's always the OCA. For big contributions, add names to the `authors` key in the manifest, for small contributions, just add to the list of contributors